src/media/js/app_list.js

```diff
@@ -12,13 +12,15 @@ define('app_list',
     }
 
     function setTrays(expanded) {
+        var $toggle = $('.app-list-filters-expand-toggle');
         // Handle tray state and populate preview thumbs.
         // Preserve the tray expand state in localStorage.
         if (expanded !== undefined) {
             expand = expanded;
         }
         $('.app-list').toggleClass('expanded', expanded);
-        $('.app-list-filters-expand-toggle').toggleClass('active', expand);
+        $toggle.toggleClass('active', expand);
+        $toggle.addClass('show');
```

Comment:
this may sound ridiculous but should we prefix this class name? it sounds like good practice